src/networking/responses/branding-response.ts

```diff
@@ -13,8 +13,17 @@ export type BrandingAppearance = {
 export type BrandingInfoResponse = {
   app_icon: string | null;
   app_icon_webp: string | null;
-  appearance: BrandingAppearance;
+  appearance: BrandingAppearance | undefined;
```

Comment:
Hmm note that if we make this nullable, it could be a breaking change for existing SDK versions. The backend should keep sending this data for existing SDKs I think.